### ISSUE TITLE
found this dataset on college education 

### ISSUE THREAD

**Issue by ilyaperepelitsa** [kaleidopop/Development_1#2]
We could test some gender stuff as well as performance of particular colleges when they receive research grants from NSF
There is also a lot of data on education internationally - can create some economic models that are more general than just US - let me know what you thing

https://catalog.data.gov/dataset/college-scorecard/resource/d43bae25-209a-4b17-affb-6ef8ff8f84ed?inner_span=True

http://datatopics.worldbank.org/Education/wDataQuery/ByExpenditures.aspx
http://databank.worldbank.org/data/reports.aspx?source=Education-Statistics:-Education-equality&preview=off
http://iresearch.worldbank.org/edattain/
http://www.cid.harvard.edu/ciddata/ciddata.html

**Comment by ilyaperepelitsa** [comment]
yeah the NSF data is in the govSpending dataset

**Comment by lynek169** [comment]
This sounds great. I like the idea of comparing it internationally too. Maybe we can choose a country that ranks higher than the US in education and another that ranks lower?

**Comment by kaleidopop** [comment]
I like the education and spending connection.

**Comment by ilyaperepelitsa** [comment]
Yeah we just need to figure out the metrics - there is a lot of stuff there and the difficulty level of statistical methods that we will have to use may be far above whatever we do in class. Let's figure out the minimum and maximum objectives and figure out how to get to the maximum with the resources that we have. I'd rather do something ridiculously cool that's way above what's required and present it as something simple but awesome.

**Comment by ilyaperepelitsa** [comment]
we can potentially look into the student debt and test its significance on quality of life. We can try to figure out how that complicated gov spending ties with college quality affect indebted students - whether the positive effect of gov spending in grants outweighs the negative of indebtedness (if we find that students who take on debt perform financially worse than the ones that don't)

**Comment by kaleidopop** [comment]
hi. i am using github.
